submissions/tavor118sn/task_memory_game/script.js

```diff
@@ -0,0 +1,124 @@
+// General stuff
+
+const GAME_BOARD = document.getElementById('gameBoard');
+let openedCards = [];
+let pairsFound = 0;
+
+const CARDS = [
+  '8-Bit',
+  'Brok',
+  'Colt',
+  'Shelly',
+];
+
+const pairsNumber = CARDS.length;
+
+
+const shuffleArray = (array) => array.sort(function () {
+  return 0.5 - Math.random()
+});
+
+
+// Generate gameboard
+
+const createCard = (elem, index) => {
+
+  const flipCard = document.createElement('div');
+  flipCard.classList.add('flip-card');
+  flipCard.dataset.cardValue = elem;
+
+  const flipInner = document.createElement('div');
+  flipInner.classList.add('flip-card-inner');
+
+  flipInner.innerHTML = `
+    <div class="front">
+      <img class="front-img" src="./img/brawl-stars-main-icon.png" alt="Brawl Icon">
+    </div>
+      <div class="back">
```

Comment:
Maybe you can use pseud element here instead?

Reply:
@AMashoshyna Could you give an example? I'm not familiar with CSS and HTML a lot. Only basis.

Reply:
Just can't imagine how `::after` can be used here

Reply:
@AMashoshyna Done